integration/hive/src/test/java/org/apache/carbondata/hive/HiveTestUtils.java

```diff
@@ -88,7 +86,6 @@ public boolean checkAnswer(ResultSet actual, ResultSet expected) throws SQLExcep
     }
     Collections.sort(expectedValuesList);Collections.sort(actualValuesList);
     Assert.assertArrayEquals(expectedValuesList.toArray(), actualValuesList.toArray());
-    Assert.assertTrue(rowCountExpected > 0);
```

Comment:
why removed this validation ?

Reply:
As now we are checking for empty table also which may contain zero row, so I have removed this.